buf.yaml

```diff
@@ -15,6 +15,7 @@ lint:
     - SERVICE_SUFFIX
     - PACKAGE_VERSION_SUFFIX
   ignore:
+    - confio
```

Comment:
Not sure exactly what caused it, but I started getting confio proto lint errors on CI: https://github.com/cosmos/cosmos-sdk/pull/6905/checks?check_run_id=948424101#step:3:18

Adding an ignore to them

Reply:
could this ignore be on third_party/proto instead of individual folders?

Reply:
I tried that, no.

Ref: https://buf.build/docs/configuration/#bufyaml-reference

```
  # All directories and file paths are specified relative to a root directory,
  # that is *with the root directory stripped*. For example, if you have a root
  # "proto" and want to ignore the directory "proto/bat", you would specify
  # "bat" in the ignore list.
  #
  # The directory "." is not allowed
```

Reply:
![](https://media2.giphy.com/media/j0qSbeNFuzjhXKFVSP/giphy.gif)